test/validate.spec.ts

```diff
@@ -1,6 +1,35 @@
 import { expect } from "chai";
 import { checkTitle } from "../src/validate";
 
+const allowedTypes = [
+  "Bugfix",
+  "Feature",
+  "Refactor",
+  "Testing",
+  "Documentation",
+  "VPAT",
+  "UI/UX",
+  "Dependency",
+  "DevDependency",
+];
+const allowedModules = [
```

Comment:
Instead of repeating these two arrays across here and `src/validate.ts`, I would just export the arrays in `src/validate.ts` to reuse them here, so that it lowers the barrier of adding new values.